packages/core/src/RPA/core/windows/helpers.py

```diff
@@ -0,0 +1,26 @@
+import os
+from typing import Dict, Union
+import psutil
+
+
+IS_WINDOWS = os.name == "nt"
+
+
+def get_process_list() -> Dict:
+    """Get process list.
+
+    Returns dictionary mapping process id to process name
+    """
+    process_list = {}
+    for proc in psutil.process_iter():
+        process_list[proc.pid] = proc.name()
+    return process_list
```

Comment:
minor: dict comprehensions are a thing, e.g. `return { proc.pid: proc.name() for proc in psutil.process_iter() }`